src/app/content/components/Page/contentLinkHandler.spec.ts

```diff
@@ -39,7 +42,7 @@ describe('contentLinkHandler', () => {
 
     beforeEach(() => {
       contentRoute = require('../../routes').content;
-      handler = require('./contentLinkHandler').contentLinkHandler(anchor, () => prop);
+      handler = require('./contentLinkHandler').contentLinkHandler(anchor, () => prop, intl);
```

Comment:
there is a testServices helper that would probably work better, i assume you're not using the spy for anything